cypress/e2e/models/migration/archetypes/archetype.ts

```diff
@@ -192,4 +196,52 @@ export class Archetype {
             }
         }
     }
+
+    perform_assessment(
+        risk,
+        stakeholders?: Stakeholders[],
+        stakeholderGroups?: Stakeholdergroups[],
+        questionnaireName = legacyPathfinder,
+        saveAndReview = false
+    ) {
+        Archetype.open();
+        clickKebabMenuOptionArchetype(this.name, "Assess");
+        cy.wait(SEC);
+        Assessment.perform_assessment(
+            risk,
+            stakeholders,
+            stakeholderGroups,
+            questionnaireName,
+            saveAndReview
+        );
+    }
+
+    validateAssessmentField(): void {
+        Archetype.open(true);
+        this.sidedrawerTab("Details");
+        cy.get(archetype.sideDrawer.risk).contains("Archetype risk");
+        cy.get(archetype.sideDrawer.riskValue).contains(/High|Medium|Low/g);
```

Comment:
Let's assert the exact value of risk from test as there was a bug that assessment was always showing "high" value .. This method can accept value of risk and assert .

Reply:
We can do that in next PR though . I will merge this one .